mobile/src/common/hooks/Dish/useDish.ts

```diff
@@ -0,0 +1,38 @@
+import {useQuery} from '@tanstack/react-query';
+import {DishResponse, Ingredient} from '../../../api/dish/types';
+import {getDish} from '../../../api/dish';
+import {useDishContext} from '../../contexts/DishContext/useDishContext';
+import {useMemo} from 'react';
+
+const useDish = (isEnabled?: boolean) => {
+  const {dishId} = useDishContext();
+  const {data, isLoading, refetch} = useQuery<DishResponse>({
+    queryKey: ['dish', dishId],
+    queryFn: () => getDish(dishId),
+    enabled: isEnabled ?? undefined,
```

Comment:
Jak nie isEnabled to jest undefined, nie ma potrzeby podstawiać undefined jeśli undefined